producer.py

```diff
@@ -36,43 +41,38 @@
 
 from thoth.common import __version__ as __common__version__
 from thoth.storages import __version__ as __storages__version__
-from thoth.python import __version__ as __python__version__
+from thoth.messaging import __version__ as __messaging__version__
+from version import __version__
 
-__version__ = "0.6.1"
-__service_version__ = f"{__version__}+\
-    storages.{__storages__version__}.\
-        common.{__common__version__}.\
-            python.{__python__version__}"
+from thoth.messaging import MessageBase
+from thoth.messaging.package_releases import PackageReleaseMessage
 
 init_logging()
+app = MessageBase().app
 
-_LOGGER = logging.getLogger("thoth.package_releases")
-_LOGGER.info(f"Thoth package-release-job v%s", __service_version__)
+_LOGGER = logging.getLogger("thoth.package_releases_job")
+__service_version__ = f"{__version__}+storages.{__storages__version__}.common.{__common__version__}.messaging.{__messaging__version__}"  # noqa: E501
+_LOGGER.info(f"Thoth-package-releases-job-producer v%s", __service_version__)
 
-_THOTH_METRICS_PUSHGATEWAY_URL = os.getenv("PROMETHEUS_PUSHGATEWAY_URL")
+prometheus_registry = CollectorRegistry()
+_THOTH_MY_NAMESPACE = os.getenv("NAMESPACE")
 _THOTH_DEPLOYMENT_NAME = os.environ["THOTH_DEPLOYMENT_NAME"]
+__COMPONENT_NAME = "thoth.package_releases_job"
```

Comment:
I think this should just be a single underscore

Reply:
thanks, https://github.com/thoth-station/package-releases-job/pull/449/commits/2192c2ce783e8463bf3d2cd9b975a212baf2b722